README.md

```diff
@@ -45,7 +45,7 @@ Announcements about GraalVM, including TruffleRuby, are made on the
 TruffleRuby is actively tested on these systems:
 
 * Ubuntu 16.04 LTS
-* macOS 10.12
+* macOS 10.13
```

Comment:
It might be useful to add the codename next to it (High Sierra).
But this is not the version tested in CI, so we consider actively tested platforms we develop on?
Then I'd like to add the Fedora versions I use.

Reply:
Right, I'll have to rethink what I say about macOS.